tests/test_tlv_parser.py

```diff
@@ -101,6 +101,30 @@ def test_parse_tlv() -> None:
     }
 
 
+def test_parse_tlv_apple() -> None:
+    """Test the TLV parser from a (truncated) dataset from an Apple BR."""
+    dataset_tlv = "0e08000065901a07000000030000194a0300000f35060004001fffc003104d79486f"
+    "6d6531323331323331323334"
```

Comment:
As pointed out by pytest, this is meaningless. Are you trying to split a long string on two lines?

Reply:
Yes :see_no_evil: Fixed.